### ISSUE TITLE
Avoid negative count values

### ISSUE THREAD

**Issue by Sergio-Muriel** [theoldreader/api#31]
Sometimes "somehow", i got negative count values calling https://theoldreader.com/reader/api/0/unread-count?all=true&output=json

The results received is:
- http://pastebin.com/t7xPBmgF

Expected value:
- no negative values, should be "count" : 0

**Comment by krasnoukhov** [comment]
Well, that's not cool. Maybe @jfiorato could help?

**Comment by jfiorato** [comment]
Yep, I'll take a look. We've seen it on the site too, just been unable to reproduce so far. I'll see what I can find.
